source/PlanetPanel.cpp

```diff
@@ -346,45 +346,47 @@ void PlanetPanel::CheckWarningsAndTakeOff()
 		// Warn about missions that will fail on takeoff.
 		if(missionCargoToSell > 0 || overbooked > 0)
 		{
-			const bool both = (missionCargoToSell > 0 && overbooked > 0);
 			out << "If you take off now, you will abort a mission due to not having enough ";
 
 			if(overbooked > 0)
 			{
 				out << "bunks available for " << overbooked;
 				out << (overbooked > 1 ? " of the passengers" : " passenger");
-				out << (both ? " and not having enough " : ".");
+				out << (missionCargoToSell > 0 ? " and not having enough " : ".\n\n");
 			}
 
 			if(missionCargoToSell > 0)
-				out << "cargo space to hold " << Format::CargoString(missionCargoToSell, "your mission cargo") << ".";
+			{
+				out << "cargo space to hold " << Format::CargoString(missionCargoToSell, "your mission cargo");
```

Comment:
```suggestion
				out << "cargo space to hold " << Format::CargoString(missionCargoToSell, "mission cargo.\n\n");
```

Instead of 'due to not having enough cargo space to hold x ton of your mission cargo', it will be 'due to not having enough cargo space to hold x ton of mission cargo.

In other places of the game, it's always written 'x ton of mission cargo' and not 'x ton of your mission cargo'.